requirements_optional.txt

```
@@ -1,3 +1,3 @@
-pillow>4.0.0,<10.0.0
+pillow>=10.0.0
```

Pillow 10.0.0 contains a high-severity vulnerability.

```suggestion
pillow>=10.0.1
```

---

Sharp eye, sir - was targeting to address that exact case, thanks!